packages/charts/react-charts/library/src/components/VerticalBarChart/__snapshots__/VerticalBarChartRTL.test.tsx.snap

```diff
@@ -10973,57 +10973,6 @@ exports[`Vertical bar chart rendering Should render the vertical bar chart with
           class="fui-Overflow fui-legend__resizableArea"
           style="text-align: unset;"
         >
-          <button
-            aria-label="undefined"
-            aria-selected="false"
-            class="fui-Button fui-legend__legend"
-            data-overflow-item=""
-            role="option"
-            style="--rect-height: 12px; --rect-backgroundColor: #627CEF; --rect-borderColor: #627CEF;"
-            type="button"
-          >
-            <div
-              class="fui-legend__rect"
-              style="height: 12px; background-color: rgb(98, 124, 239); border-color: #627cef; --rect-content-high-contrast: linear-gradient(to right, #627CEF, #627CEF);"
-            />
-            <div
-              class="fui-legend__text"
-            />
-          </button>
-          <button
-            aria-label="undefined"
-            aria-selected="false"
-            class="fui-Button fui-legend__legend"
-            data-overflow-item=""
-            role="option"
-            style="--rect-height: 12px; --rect-backgroundColor: #C19C00; --rect-borderColor: #C19C00;"
-            type="button"
```

Comment:
Why did elements get removed in existing snapshot. Were we showing duplicate elements already without the histogram scenario